docs/install_guides/installing-openvino-conda.md

```diff
@@ -1,10 +1,118 @@
-# Install OpenVINO™ Runtime from Anaconda Cloud
+# Install OpenVINO™ Runtime from Anaconda Cloud {#openvino_docs_install_guides_installing_openvino_conda}
 
-* [Install OpenVINO Runtime from an Archive File](installing-openvino-from-archive-linux.md)
-* [Install OpenVINO from PyPI](installing-openvino-pip.md)
-* [Install OpenVINO with Docker](installing-openvino-docker-linux.md)
-* [Build From Source](https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/build.md)
+@sphinxdirective
 
-The other installation methods are temporarily unavailable.
+.. note:
+
+   Installing OpenVINO Runtime from Anaconda Cloud is recommended for C++ developers, as it provides only the C++ Runtime API.
+
+   If you work with Python, the PyPI package includes everything you need to develop and deploy OpenVINO. 
+   Visit the :doc:`Install OpenVINO from PyPI <openvino_docs_install_guides_installing_openvino_pip>` page for instructions 
+   on how to install OpenVINO Runtime for Python using PyPI.
+
+.. tab:: System Requirements
+
+  | Full requirement listing is available in:
+  | `System Requirements Page <https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/system-requirements.html>`__
+
+.. tab:: Processor Notes
+
+  | Not all Intel CPUs include integrated graphics processors.  
```

Comment:
Not sure if it is relevant at all, but just to note - current conda package does not provide GPU support.

Reply:
This is hugely relevant! :) I'll add it in the disclaimer at the top.

Reply:
For 2023.0 GPU support is still an open question. 
We still have an ability to add GPU plugin for 2023.0

But if we do it, let's update the page later one more time.

Reply:
So, if this package has no support for GPU, I'll remove this line. In this case, it does not matter if a CPU has integrated graphics or not... And in the future, we'll bring the line back.